content/zh/blogs/deep-dive-into-the-K8s-request-and-limit.md

```diff
@@ -0,0 +1,256 @@
+---
+title: '深入理解 K8s 中的 request 和 limit'
+tag: 'Kubernetes,KubeSphere,schedule,monitoring'
+createTime: '2021-01-01'
+author: '饶云坤'
+snapshot: ''
+---
+
+放眼全球， IT 产业在过去的 20 年里取得了令人瞩目的长足进步与飞速发展，网络数据几乎是以指数的形式膨胀，并且深刻地影响着我们每一个人日常。当然也对整个 IT 产业的顶层技术架构不管是在计算，存储还是网络都提出了更高的要求。在这期间，云的概念也开始普及，许多公司都部署了实施云化战略，纷纷搭建起云平台，希望完成传统应用到云端的迁移。但是这个过程中也遇到了一些问题诸如敏捷性，可扩展性等问题。而近几年来提出的云原生（ Cloud Native ）概念，近乎完美地解决了上述云化过程中遇到的这些问题。而具有 Google 背书和相关容器生态的容器编排系统， 作为一个事实上的标准， Kubernetes 很自然的走入了人们的视野。但是 Kubernetes 天然具有比较陡峭的学习曲线，缺乏周边软件配套。而 KubeSphere 作为国内唯一一个开源的 Kubernetes（ K8s ）发行版，极大地降低了使用 Kubernetes 的门槛，所以其一经出现，便广泛收到开源社区和相关上下游企业的广泛好评。越来越多的企业选择通过 KubeSphere 在生产环境的 K8s 集群上部署应用。
+
```

Comment:
I suggest do not advertise ks here. This blog is talking about K8s request and limit, not related to ks actually. You just use ks to demo how easily to set and monitor these metrics.

Reply:
Got it